Mirror/Runtime/NetworkTransformChild.cs

```diff
@@ -298,23 +298,9 @@ void FixedUpdateClient()
             // interpolate on client
             if (m_LastClientSyncTime != 0)
             {
-                if (m_InterpolateMovement > 0)
-                {
-                    m_Target.localPosition = Vector3.Lerp(m_Target.localPosition, m_TargetSyncPosition, m_InterpolateMovement);
-                }
-                else
-                {
-                    m_Target.localPosition = m_TargetSyncPosition;
-                }
+                m_Target.localPosition = m_InterpolateMovement > 0 ? Vector3.Lerp(m_Target.localPosition, m_TargetSyncPosition, m_InterpolateMovement) : m_TargetSyncPosition;
```

Comment:
seems hard to read,  how about combining both ifs so that they if movement > 0 then we set the localPostion and rotation?

Reply:
what do you mean by combining?

Reply:
```cs
if (m_InterpolateMovement > 0) {
    m_Target.localPosition = Vector3.Lerp(m_Target.localPosition, m_TargetSyncPosition, m_InterpolateMovement);
    m_Target.localRotation = Quaternion.Slerp(m_Target.localRotation, m_TargetSyncRotation3D, m_InterpolateRotation);
}
else 
{
    m_Target.localPosition = m_TargetSyncPosition;
    m_Target.localRotation = m_TargetSyncRotation3D;
}
```